multifidelity_end2end.py

```diff
@@ -636,16 +672,22 @@ def add_args(parser: ArgumentParser):
             "delta_ml",
             "trad_delta_ml",
             "transfer",
+            "evidentialmf",
+            "evidentialdual",
+            "mvemultifidelity",
+            "kernel"
         ],
     )
-    parser.add_argument("--data_file", type=str, default="tests/data/gdb11_0.001.csv")
-    # choices=["multifidelity_joung_stda_tddft.csv", "gdb11_0.0001.csv" (too small), "gdb11_0.0001.csv"]
     parser.add_argument(
-        "--hf_col_name", type=str, default="h298"
-    )  # choices=["h298", "lambda_maxosc_tddft"]
+        "--data_file", type=str, default="/home/temujin/chemprop-mf/tests/data/lambda.csv"
```

Comment:
use relative path rather than absolute, the data file default can stay the same as it was before